server/models/procedures.sql

```diff
@@ -1304,4 +1304,10 @@ BEGIN
   SELECT DISTINCT BUID(record_uuid) AS record_uuid, trans_id, code FROM stage_trial_balance_errors ORDER BY code, trans_id;
 END $$
 
-DELIMITER ;
+-- compute account class
+CREATE PROCEDURE ComputeAccountClass()
```

Comment:
Is this procedure ever called from the code?   If not, it probably should be in `admin.sql`.